scss/bootstrap/overrides/style/_button-groups.scss

```diff
@@ -0,0 +1,22 @@
+.btn .caret {
+  margin: 0 5px;
+}
+
+.btn-group {
+
+  .btn {
+    box-shadow: none;
+    border-color: $input-border;
+
+    &:not([class*=" btn-"]) {
+      &:hover {
+        border-color: $input-border;
+      }
+    }
+
+    &.active {
+      background-color: $gray;
+      color: #FFF;
```

Comment:
This file is reused in SSP and Shoreditch, and in both these themes, the variables for #FFF is different, that is the reason I had to remove the variable and use the color directly. Otherwise compilation is failing.